storage/README.md

```diff
@@ -2,13 +2,58 @@
 
 The Storage API provides a simple key/value persistent store for lightweight data.
 
+Mobile OS's may periodically clear data set in `window.localStorage`, so this
+API should be used instead. This API will fall back to using `localStorage`
+when running as a Progressive Web App.
+
+This plugin will use
+[`UserDefaults`](https://developer.apple.com/documentation/foundation/userdefaults)
+on iOS and
+[`SharedPreferences`](https://developer.android.com/reference/android/content/SharedPreferences)
+on Android. Stored data is cleared if the app is uninstalled.
+
+**Note**: this API is _not_ meant for high-performance data storage
```

Comment:
I think another word should be used instead of high-performance, because I've seen people thinking this means slow, which is not true.